packages/cli/src/CliUtils.ts

```diff
@@ -116,24 +120,45 @@ export const showMessage = (
 
 export const getMetaplexInstance = (
   connection: Connection,
-  keypair: Keypair
+  keypair: Keypair,
+  storageParams: string = ""
 ) => {
   const metaplex = Metaplex.make(connection).use(keypairIdentity(keypair));
   const isDevnet = connection.rpcEndpoint.includes("devnet");
 
-  const bundlrStorageDriver = isDevnet
-    ? new BundlrStorageDriver(metaplex, {
+  //TODO: Use DI for this
+  const s3Mgr = new S3StorageManager(new EnvVariables());
+  s3Mgr.parseCmdArg(storageParams);
+
+  if (s3Mgr.hasS3Config) {
+    const awsClient = new S3Client({
+      region: "us-east-1",
```

Comment:
Should region be specified as part of the input? It seems like, if we allow specifying `bucket` at the CLI, we should specify `region` there too